agent/lib/kontena/workers/node_info_worker.rb

```diff
@@ -171,5 +172,10 @@ def calculate_memory
     def docker_info
       @docker_info ||= Docker.info
     end
+
+    # @return [String]
+    def version
+      @version ||= File.read(File.realpath(File.join(__dir__, '../../../VERSION'))).strip
```

Comment:
No need to parse this here.. it should be available from `Kontena::Agent::VERSION`